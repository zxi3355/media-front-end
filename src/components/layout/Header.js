import React from 'react';

const Header = () => {
  return (
    <header>
      <nav className='navbar fixed-top navbar-light bg-light'>
        <div className='container-fluid'>
          <a className='navbar-brand' href='#'>
            Article Feed
          </a>
        </div>
      </nav>
    </header>
  );
};

export default Header;
