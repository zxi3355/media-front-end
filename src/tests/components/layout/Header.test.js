import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../../components/layout/Header';

test('Should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  const text = wrapper.find('.navbar-brand').text();
  expect(text).toEqual('Article Feed');
  expect(toJson(wrapper)).toMatchSnapshot();
});
