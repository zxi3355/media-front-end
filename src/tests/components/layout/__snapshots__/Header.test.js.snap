// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header correctly 1`] = `
<header>
  <nav
    className="navbar fixed-top navbar-light bg-light"
  >
    <div
      className="container-fluid"
    >
      <a
        className="navbar-brand"
        href="#"
      >
        Article Feed
      </a>
    </div>
  </nav>
</header>
`;
